### PR TITLE
Save Image object to Parse with AWS url

### PR DESCRIPTION
**Pull request by sukhada**
@caro5

### CODE REVIEW

File: app/src/main/java/turtler/voyageur/activities/BaseActivity.java [hunk 1]
```diff
@@ -268,6 +327,55 @@ public void onError(int i, Exception e) {
         }
     }
 
+    public void saveImageToParse(Location mLastLocation, File resizedFile) {
+        String lat = Double.toString(mLastLocation.getLatitude());
+        String lon = Double.toString(mLastLocation.getLongitude());
+        ParseObject parseImage = ParseObject.create("Image");
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/activities/BaseActivity.java]: Can you fix the Image model class, and create getters and setters, constructor if needed. We should be able to do:
Image image = new Image();
image.setLatitude(lat);
etc

**Comment by sukhada** [comment]
@caro5 updated

File: app/src/main/java/turtler/voyageur/models/Image.java [hunk 2]
```diff
@@ -36,6 +36,26 @@ public double getLongitude() {
 
     public Image() {}
 
+    public void set_picture_url(String picture_url) {
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/models/Image.java]: setPictureUrl and pictureUrl

File: app/src/main/java/turtler/voyageur/models/Image.java [hunk 3]
```diff
@@ -36,6 +36,26 @@ public double getLongitude() {
...
+
+    public void setLatitude(double latitude) {
+        this.latitude = latitude;
+    }
+
+    public void setLongitude(double longitude) {
+        this.longitude = longitude;
+    }
+
     public Image(String picture_url, User user, Event event, double latitude, double longitude) {
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/models/Image.java]: pictureUrl

**Comment by sukhada** [comment]
@caro5 updated 4real

File: app/src/main/java/turtler/voyageur/models/Image.java [hunk 4]
```diff
@@ -28,16 +34,37 @@ public double getLongitude() {
...
+    public void setPictureUrl(String pictureUrl) {
+        put(PICTURE_URL_KEY, pictureUrl);
+    }
+
+    public void setUser(User user) {
+        put(USER_KEY, user);
+    }
+
+    public void setEvent(Event event) {
+        this.event = event;
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/models/Image.java]: this should be put too

Comment (sukhada) [reply app/src/main/java/turtler/voyageur/models/Image.java]: yeah, just haven't implemented it yet but i can update

File: app/src/main/java/turtler/voyageur/models/Image.java [hunk 5]
```diff
@@ -8,8 +8,14 @@
...
+    public static final String LONGITUDE_KEY = "longitude";
+    public static final String USER_KEY = "user";
+    public static final String PICTURE_URL_KEY = "pictureUrl";
+
+
+    public String getPictureUrl() {
+        return pictureUrl;
     }
 
     public User getUser() {
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/models/Image.java:22]: return (User) getParseUser(USER_KEY);

and for Event,
return (Event) getParseObject(EVENT_KEY);

File: app/src/main/java/turtler/voyageur/models/Image.java [hunk 6]
```diff
@@ -8,8 +8,14 @@
...
-    public String getPicture_url() {
-        return picture_url;
+    public static final String LATITUDE_KEY = "latitude";
+    public static final String LONGITUDE_KEY = "longitude";
+    public static final String USER_KEY = "user";
+    public static final String PICTURE_URL_KEY = "pictureUrl";
+
+
+    public String getPictureUrl() {
+        return pictureUrl;
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/models/Image.java]: return getString(PICTURE_URL_KEY);

**Comment by caro5** [comment]
lgtm 🍨

**Pull request merged**